packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -425,10 +425,9 @@ describe('ReactCompositeComponent', () => {
     expect(() => {
       instance = ReactDOM.render(<Component />, container);
     }).toWarnDev(
-      'Cannot update during an existing state transition (such as within ' +
```

Comment:
Can we add "(such as within render...)" part back? I don't see why it needs to be changed.

Reply:
was just thinking it was lengthy.
 maybe perhaps instead of printing "Cannot update during an existing state transition" ,  It is better to be concise and print "Cannot update state during render lifecycle method" ?